content/02.intro.md

```diff
@@ -36,30 +37,38 @@ Easy-to-use software packages have brought the techniques of the field out of th
 Additionally, new techniques for fast training have enabled their application to larger datasets [@arxiv:1106.5730].
 Dropout of nodes, edges, and layers makes networks more robust, even when the number of parameters is very large.
 New neural network approaches are also well-suited for addressing distinct challenges.
-For example, neural networks structured as autoencoders or as adversarial networks require no labels and are now regularly used for unsupervised tasks.
-In this review, we do not exhaustively discuss the different types of deep neural network architectures; an overview of the principal terms used herein is given in Table @tbl:glossary.
-A recent book from Goodfellow et al. covers these in detail [@url:http://www.deeplearningbook.org/].
 Finally, the larger datasets now available are also sufficient for fitting the many parameters that exist for deep neural networks.
 The convergence of these factors currently makes deep learning extremely adaptable and capable of addressing the nuanced differences of each domain to which it is applied.
 
+![Neural networks come in many different forms. Left: a key for the various types of nodes used in neural networks. Simple FFNN: a feed forward neural networks in which inputs are connected via some function to an output node and the model is trained to produce some output for a set of inputs. MLP: the multi-layer perceptron is a feed forward neural network in which there is at least one hidden later between the input and output nodes. CNN: the convolutional neural network is a feed forward neural network in which the inputs are grouped spatially into hidden nodes. In the case of this example, each input node is only connected to hidden nodes alongside their neighboring input node. Deep autoencoder: a type of MLP in which the neural network is trained to produce an output that matches the input to the network. Deep RNN: a deep recurrent neural network is used to allow the neural network to retain memory over time or sequential inputs. ](https://user-images.githubusercontent.com/542643/34907749-3115860a-f851-11e7-99a7-03d4d8a27802.png){#fig:nn-petting-zoo}
```

Comment:
Can we make this caption one sentence per line?

`feed forward neural networks` -> `feed forward neural network`

`hidden later` -> `hidden layer`

I'm confused about the CNN diagram and caption.  Is this something like a 1D CNN with a width of 2 input units?  The first and third input node connect to the middle hidden unit, suggesting that I'm misreading something.

Is there something special about the Autoencoder and RNN that we want to convey with `Deep` in the label?  Or should we drop `Deep`?

Where is the image hosted?  Is there a way we can modify it in the future?  I'm not sure what the best way to track the source in the Google Drawings page would be.  Maybe we should export svg and png versions and add them to this pull request.  I'm not sure whether we could later import the svg into Google Drawings to edit it again or if that is even a concern for us.

Reply:
Image is from this discussion and is hosted by github:
https://github.com/greenelab/deep-review/issues/684

I'll make this 1 sentence per line. Wasn't sure if that'd work with figure captions.

Reply:
@cgreene should we also acknowledge in the caption that this figure is adapted from http://www.asimovinstitute.org/neural-network-zoo/ ?

Reply:
@agitter : yes! Added 👍